src/LandIce/evaluators/LandIce_EffectivePressure_Def.hpp

```diff
@@ -77,6 +79,34 @@ postRegistrationSetup(typename Traits::SetupData /* d */,
   }
 }
 
+// *********************************************************************
+// Kokkos functor
+template<typename EvalT, typename Traits, bool IsStokes, bool Surrogate>
+KOKKOS_INLINE_FUNCTION
+void EffectivePressure<EvalT, Traits, IsStokes, Surrogate>::
+operator() (const Surrogate_Tag& tag, const int& sideSet_idx) const {
+
+  const ParamScalarT alpha = Albany::convertScalar<const ParamScalarT>(alphaParam(0));
+
+  for (int pt=0; pt<numPts; ++pt) {
+    // N = P_o-P_w
+    N (sideSet_idx,pt) = (1-alpha)*P_o(sideSet_idx,pt);
+  }
+
+}
+
+template<typename EvalT, typename Traits, bool IsStokes, bool Surrogate>
+KOKKOS_INLINE_FUNCTION
+void EffectivePressure<EvalT, Traits, IsStokes, Surrogate>::
```

Comment:
This is fine, but for the record, given the small size, I believe keeping both cases in a single kernel (with an if to determine which for loop to execute) would have been good enough. The branch predictor will likely hit the correct branch.